mitosheet/mitosheet/code_chunks/export_to_file_code_chunk.py

```diff
@@ -10,6 +10,33 @@
 from mitosheet.state import State
 from mitosheet.transpiler.transpile_utils import TAB, column_header_to_transpiled_code
 
+# This is a helper function that generates the code for formatting the excel sheet
+def get_format_code(state: State) -> list:
+    code = []
+    formats = state.df_formats
+    for sheetIndex in range(len(formats)):
+        sheet_name = state.df_names[sheetIndex]
+        format = formats[sheetIndex]
+    
+        # If there is no formatting, we skip trying to access the colors
+        if format.get('headers') is None:
+            continue
+        
+        # Prepares the header font color and background color to be used in the API call
+        # So either they should be surrounded by quotes, or they should be None
+        header_font_color = None
+        header_background_color = None
+        if (format['headers'].get('color') is not None):
+            header_font_color = f'"{format["headers"]["color"]}"'
+        if (format['headers'].get('backgroundColor') is not None):
+            header_background_color = f'"{format["headers"]["backgroundColor"]}"'
+        
+        # If both are None, we skip this sheet
+        if header_font_color is None and header_background_color is None:
+            continue
+        code.append(f'{TAB}add_formatting_to_excel_sheet(writer, "{sheet_name}", {header_background_color}, {header_font_color})')
```

Comment:
I'd love if this code transpiled as:
```
add_formatting_to_excel_sheet(writer, "sheet name", header_background_color="00000", header_font_color="XXXXXX")
```

That way, users are able to tell which color is which when they are editing the code. I'm also just thinking that it will:
1. Make it much easier to read as we add more formatting options. 
2. Allow us to reorder the parameters if we want to do so in the function, without breaking users code. 

Let me know what what you think! This is where the `param_dict_to_code` utility could be useful -- it's in `transpile_utils.py` -- check it out!

Reply:
Oh yeah that makes sense! I'm a little confused about how using `param_dict_to_code` would work here - does the code I just pushed look ok?